package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "terra-enzyme-intl",
+  "version": "0.0.0",
+  "description": "Enzyme helpers for fortifying tests that depend on react-intl by decoupling the need for actual translations.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-enzyme-intl.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-enzyme-intl",
+    "jest",
+    "enzyme",
+    "react",
+    "react-intl",
+    "i18n",
+    "intl"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/cerner/terra-enzyme-intl#readme",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-enzyme-intl/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "browserslist-config-terra": "^1.1.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "danger": "^3.7.14",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.10.0",
+    "enzyme-to-json": "^3.3.5",
+    "eslint": "^5.14.1",
+    "eslint-config-terra": "^2.4.0",
+    "jest": "^24.1.0",
+    "react": "^16.8.3",
+    "react-dom": "^16.8.3",
+    "react-intl": "^2.8.0"
+  },
+  "peerDependencies": {
+    "enzyme": "3.x",
+    "react": "16.x",
```

Comment:
Technically this could be opened up to `"react": "^0.14.9 || ^15.0.0 || ^16.0.0"` based on `react-intl` react dependency. Since this is a toolkit setup, it might be with opening this up (along with enzyme) incase teams would like to utilize this with react 15....

This might not be something we want to do but a thought. It would be more to maintain. @dev-cprice Is there anything in this setup that is react 16 specific?

Reply:
There's nothing that's react 16 specific and I could see wanting to open it up, but is it worth it? I don't have a strong opinion either way.

Reply:
We don't really test or support ReactJS versions lower than 16, I don't think we should differ compared to what we support in our terra components for this repo. I'd leave it as is.